go-server-server/go/persistent.go

```diff
@@ -291,6 +302,30 @@ func SwssGetVrouterRoutes(vnet_id_str string, vnidMatch int, ipFilter string) (r
         routes = append(routes, routeModel)
     }
 
+    for k, kvp := range kv2 {
+        ipprefix := strings.Split(k, db.separator)[2]
+
+        routeModel := RouteModel{
+            IPPrefix:    ipprefix,
+            NextHop:     kvp["nexthop"],
+        }
+
+        if vnid, ok := kvp["vni"]; ok {
```

Comment:
You can remove the vnid section from this case as this is not part of this route structure